readthedocs/doc_builder/exceptions.py

```diff
@@ -42,6 +42,13 @@ class BuildUserError(BuildBaseException):
     )
 
 
+class BuildUserSkip(BuildUserError):
+    message = gettext_noop(
+        "This build was cancelled due the magic exit code was returned by a commmand."
```

Comment:
We probably don't want to describe this as "magic", instead it's just a feature we support

```suggestion
        "This build was manually skipped using a command exit code."
```